netty/src/main/java/io/grpc/transport/netty/NettyClientTransport.java

```diff
@@ -266,24 +228,6 @@ public void shutdown() {
     }
   }
 
-  /**
-   * Waits until started. Does not throw an exception if the transport has now failed.
-   */
-  private void awaitStarted() {
-    if (!started) {
-      try {
-        synchronized (this) {
-          while (!started) {
-            wait();
-          }
-        }
-      } catch (InterruptedException ex) {
-        Thread.currentThread().interrupt();
-        throw new RuntimeException("Interrupted while waiting for transport to start", ex);
-      }
-    }
-  }
-
   private synchronized void notifyStarted() {
```

Comment:
This can be removed, and the calling of it in notifyShutdown(), and the started variable itself.